geopandas/tests/test_geoseries.py

```diff
@@ -322,6 +322,19 @@ def test_to_wkb(self):
     def test_to_wkt(self):
         assert_series_equal(pd.Series([self.t1.wkt, self.sq.wkt]), self.g1.to_wkt())
 
+    def test_from_xy_points(self):
+        x = np.array([pt.x for ind, pt in self.landmarks.iteritems()])
+        y = np.array([pt.y for ind, pt in self.landmarks.iteritems()])
+        index = self.landmarks.index.tolist()
+        crs = self.landmarks.crs
+        assert_series_equal(self.landmarks, GeoSeries.from_xy(x, y, index=index, crs=crs))
```

Comment:
```suggestion
        assert_geoseries_equal(self.landmarks, GeoSeries.from_xy(x, y, index=index, crs=crs))
```

we need our assertion to check all geometry specific attributes